src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/EnumVariantViewOfEnumerator.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections;
+using System.Runtime.InteropServices.ComTypes;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    [Serializable]
```

Comment:
Does this need to be [Serializable] for a particular scenario?  We've been trying to be very explicit about what types we make [Serializable] in core.
cc: @ViktorHofer

Reply:
I don't think they need to be marked `[Serializable]` for anything in particular wrt the scenario this dll enables. I'll remove them.